cmd/to_base64/main.go

```diff
@@ -17,7 +18,12 @@ func main() {
 
 func toBase64(element string) string {
 	if cli.HexRegexp.MatchString(element) {
+		if strings.HasPrefix(element, "0x") {
```

Comment:
Safe to do this here because the value is for sure a hexadecimal (look previous line).